### DISCUSSION TITLE
Update the Debian Bullseye docker image

### DISCUSSION THREAD

**Discussion by Thovi98** [photoprism/photoprism#3123] [General]
Hello dear devs ! Thanks a lot for all your work !!

Since a few month now, Photoprism is packaged for [Yunohost ](https://yunohost.org/#/) (an os wich tries to ease as much as possible the install and the maintenance of a server). 
The latest version of Yunohost is based on Debian Bulsseye. 

The latest version of the [Yunohost Photoprism’s package](https://github.com/YunoHost-Apps/photoprism_ynh) is based on the build from 01 september 2022. When I try to update to the latest version (from 18 november 2022), I have the following `glibc` error:

```
/var/www/photoprism/live/bin/photoprism: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /var/www/photoprism/live/bin/photoprism)
/var/www/photoprism/live/bin/photoprism: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /var/www/photoprism/live/bin/photoprism)
/var/www/photoprism/live/bin/photoprism: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /var/www/photoprism/live/bin/photoprism)
```

If I understand well, those `glibc `versions are not available for Bullseye and since the latest docker images are made from (and for) Jammy, they are made with glibc v.2.32 or higher.

So, do you think it would be possible to update the Bullseye docker builds to the latest version of Photoprism? Or have you a suggestion to bypass this error?

Appart from that, everything works well: Photoprism is a really amazing app 🤩

Thanks for your reading and input.

**Comment by lastzero** [comment]
We no longer actively use or promote the older Docker images because they contain bugs that are only fixed in our latest image. This is to reduce the number of support requests we receive. However, the builds should generally still work and we are happy to accept pull requests for them.

**Reply by lastzero** [reply]
To be clear, I do not mean bugs in PhotoPrism but in distribution packages such as Darktable, Rawtherapee, libheif, and FFmpeg.

**Reply by Thovi98** [reply]
Thanks for your fast answer!
I don’t know much of all those things work unfortunately. 

If somebody who reads this wants to try to update the Bullseye docker image, don’t hesitate :)

**Reply by lastzero** [reply]
One way would be to build those dependencies from source or install backports if available. Unfortunately, our team has no time to do this.

**Reply by Thovi98** [reply]
I understand, thanks anyway for all you do for this awesome app! :)
